Content.Server/Chemistry/EntitySystems/ChemistrySystemHypospray.cs

```diff
@@ -99,7 +99,7 @@ public bool TryDoInject(EntityUid uid, EntityUid? target, EntityUid user, Hyposp
 
             if (target != user)
             {
-                _popup.PopupEntity(Loc.GetString("hypospray-component-feel-prick-message"), target.Value);
+                _popup.PopupEntity(Loc.GetString("hypospray-component-feel-prick-message"), target.Value, target.Value);
```

Comment:
Does it really needs the `Value` attribute ? Isn't `target` enough ?

Reply:
not enough, its nullable